application/web/index.html

```diff
@@ -95,6 +95,13 @@
                 <!-- Temporarily put textarea for the CodeMirror editor -->
                 <textarea class="entry-textarea" placeholder="Enter your thoughts here"></textarea>
                 <div class="past-entries"></div>
+
+                <div id="no-entry-text">
+                    <strong>You have no Journals</strong>
```

Comment:
you might wanna use <b> instead of <strong> in this case since the phrase "you have no journals" doesn't have much semantic meaning in the topic of our project and the internet